contracts/cw20-atomic-swap/Cargo.toml

```
@@ -18,16 +18,16 @@ backtraces = ["cosmwasm-std/backtraces"]
 library = []
 
 [dependencies]
-cw-utils = { version = "0.12.1" }
-cw2 = { version = "0.12.1" }
-cw20 = { version = "0.12.1" }
-cosmwasm-std = { version = "1.0.0-beta5" }
-cw-storage-plus = { version = "0.12.1" }
+cw-utils = { version = "0.13.2" }
+cw2 = { version = "0.13.2" }
+cw20 = { version = "0.13.2" }
+cosmwasm-std = { version = "1.0.0-beta8" }
+cw-storage-plus = { version = "0.13.2" }
```

You could simplify this for consistency.
```suggestion
cw-utils = "0.13.2"
cw2 = "0.13.2"
cw20 = "0.13.2"
cosmwasm-std = "1.0.0-beta8"
cw-storage-plus = "0.13.2"
```
in other Cargo.toml files as well.